src/components/QuickOpenModal.js

```diff
@@ -162,21 +167,26 @@ export class QuickOpenModal extends Component<Props, State> {
 
   updateResults = (query: string) => {
     if (this.isGotoQuery()) {
-      return;
+      return this.setState({ isLoading: false });
     }
 
     if (query == "") {
-      return this.showTopSources();
+      this.showTopSources();
+      return this.setState({ isLoading: false });
```

Comment:
`this.setState({ isLoading: false });` can be moved  to the top of this function (since it always happens) so that we don't have it repeating in each if statement

Reply:
I'm glad you said this because it made me re examine my code, and realized I was causing wasteful re renders, I moved the isLoading: false to the instances when results is already being set through setState